Gruntfile.js

```diff
@@ -179,7 +179,11 @@ module.exports = function (grunt) {
     wiredep: {
       app: {
         src: ['<%= yeoman.app %>/index.html'],
-        exclude: ['bower_components/bootstrap-sass-official/assets/javascripts/bootstrap.js'],
+        exclude: [
+            'bower_components/bootstrap-sass-official/assets/javascripts/bootstrap.js',
+            // Leaflet provided by cartodb.js
+            'bower_components/leaflet-dist/*'
```

Comment:
Right.  In addition to `leaflet-dist` there is a `leaflet` bower package that some Leaflet plugins depend on, that might also need to be ignored in future if we were to bower-install other plugins.  Just a heads up.